src/pretix/base/datasync/datasync.py

```diff
@@ -252,9 +252,15 @@ def get_field_value(self, inputs, mapping_entry):
         except KeyError:
             with language(self.event.settings.locale):
                 raise SyncConfigError([_(
-                    'Field "{field_name}" is not valid for {available_inputs}. Please check your {provider_name} settings.'
-                ).format(key=key, available_inputs="/".join(inputs.keys()), provider_name=self.display_name)])
-        input = inputs[field.required_input]
+                    'Field "{field_name}" does not exist. Please check your {provider_name} settings.'
+                ).format(field_name=key, provider_name=self.display_name)])
+        try:
+            input = inputs[field.required_input]
+        except KeyError:
+            with language(self.event.settings.locale):
+                raise SyncConfigError([_(
+                    'Field "{field_name}" requires {required_input}, but got only {available_inputs}. Please check your {provider_name} settings.'
+                ).format(field_name=key, required_input=field.required_input, available_inputs=",".join(inputs.keys()), provider_name=self.display_name)])
```

Comment:
Not sure if we want it more „naturally readable“ vs. „technical“, but maybe join with an additional space:
```suggestion
                ).format(field_name=key, required_input=field.required_input, available_inputs=", ".join(inputs.keys()), provider_name=self.display_name)])
```